### PR TITLE
Separate flag for PDB enablement

### PR DESCRIPTION
**Pull request by AndyJiang99**

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -80,8 +80,10 @@ autoscaling:
   targetCPUUtilizationPercentage: 40
   # targetMemoryUtilizationPercentage: 80
 
+# Pod disruption budget flag since PodDisruptionBudget doesn't take enabled
```
Comment (aianus) [inline helm/values.yaml:83]: Just describe the purpose of the flag, no need for reasoning/implementation details

File: helm/templates/pod_disruption_budget.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{{- if .Values.podDisruptionBudget.enabled -}}
+{{- if .Values.createPodDisruptionBudget -}}
```
Comment (taha-au) [inline helm/templates/pod_disruption_budget.yaml:1]: let's simply use `{{- if .Values.podDisruptionBudget -}}` here

File: helm/values.yaml [hunk 3]
```diff
@@ -80,9 +80,12 @@ autoscaling:
...
 
-podDisruptionBudget:
-  enabled: false
-  minAvailable: 1
+# Set createPodDisruptionBudget to true and uncomment
+# podDisruptionBudget block below to enable pod disruption budget
+createPodDisruptionBudget: false
+
+# podDisruptionBudget:
+#   minAvailable: 1
```
Comment (taha-au) [inline helm/values.yaml:88]: Let's change this to the following instead of introducing a new value

```
podDisruptionBudget
#  minAvailable: 1
```

**Review by taha-au — CHANGES_REQUESTED** [review]

File: helm/values.yaml (hunk 3 above)
Comment (AndyJiang99) [reply helm/values.yaml]: Did you mean it to be 
`podDisruptionBudget:` or `podDisruptionBudget`

**Review by taha-au — APPROVED** [review]

**Pull request merged**